thirdparty/games_are_not_uwp.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/thirdparty/README.md b/thirdparty/README.md
+index dd5184a5ce..c4be92d171 100644
+--- a/thirdparty/README.md
++++ b/thirdparty/README.md
+@@ -276,6 +276,8 @@ Files extracted from upstream source:
+ - the `src` folder
+ - `AUTHORS`, `COPYING`, `THANKS`
+ 
++Applied the patch in `patches/games_are_not_uwp.patch` (upstream PR: https://github.com/harfbuzz/harfbuzz/pull/4109).
++
+ 
+ ## icu4c
+ 
```

Comment:
This part should not be in the patch.

Reply:
I messed up the patch file. Happens sometimes... In any case, they've already included my patch in 7.0.0, so I'm including a commit to move to that version in my next push.